_posts/2018-03-15-jupyterhub-binder-convergence.md

```diff
@@ -165,23 +171,34 @@ The form consists of:
 
 ## Behavior upon following a server URL/badge:
 
-.  Display the authentication page (if not yet authenticated)
+-  Display the authentication page (if not yet authenticated)
 
-.  Display a security confirmation dialog as above (if origin is not within the
+-  Display a security confirmation dialog as above (if origin is not within the
     jupyterhub), with a description of the configuration and origin.
 
-.  As above after clicking "Launch"
+-  As above after clicking "Launch"
 
 # Some use cases
 
-## Local binder (better name?)
+## Local binder (better name? [Binder@home?])
+
+Scenarios:
+
+- Luc, a researcher, discovered a nice computing environment
+  on Binder. Maybe a notebook demonstrating an interesting workflow to
+  analyze data. He wants to use it more intensively on his own data.
 
-Scenario: Luc, a researcher, discovered a nice computing environment
-on Binder. Maybe a notebook demonstrating an interesting workflow to
-analyze data. He wants to use it more intensively on his own data.
+- Lucy has found a notebook & binder environment published
```

Comment:
Is the name Luc or Lucy?

Reply:
Two different people, for parity :-)
Thanks for pointing this out.